packages/app/test/cypress/integration/10-install/install.spec.ts

```diff
@@ -53,7 +53,7 @@ context('Installing', () => {
     cy.getByTestid('btnSubmit').click();
 
     cy.screenshot(`${ssPrefix}-installed`, {
-      blackout: ['#grw-sidebar-contents-wrapper'],
+      blackout: ['#grw-sidebar-contents-wrapper','[data-line="2"]:eq(0) > a > img', '[data-hide-in-vrt="true"]'],
```

Comment:
I think the reason why `"[data-hide-in-vrt=true]"` in `/support/screenshot.ts` don't work is because the value is not correct.
Compare to the one you wrote which I believe is correct, and you will know why.